src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable camelcase */
```

Comment:
It's necessary and used in all input schemas. Needed here because I had to use an input schema in this transformer

Reply:
This is something I've been wondering. Why is it that the input for the transformers generally have camel-cased keys (I would expect snake-cased due to the format of the export data), and why is it snake-cased in this instance?

Reply:
input schemas are always `snake-cased` while transformers and outputs are mostly `camel-cased`.
In this case, since this instance I'm dealing with is a revolving entity which is the same as the ancestor. Previously we were returning `null` but we noticed that it created problems because it still needs a few other fields in the liquid template. Since the revolving entity returned uses raw data, we need to process it like a whole transformer would do and the field required in this case is `fieldRelatedLinks` which is an array of objects and we don't have an easy helper to get all the data, therefore I created an input schema to facilitate transforming the data for the output schema. I hope this make sense.